api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -318,17 +318,10 @@ public void removeUserFromGroup(String email, String groupName) {
   }
 
   @Override
-  public boolean isUserMemberOfGroup(String groupName) {
+  public boolean isUserMemberOfGroup(String email, String groupName) {
     return retryHandler.run((context) -> {
-      // There is no endpoint in FireCloud for checking whether a user is a member of a particular
-      // group; so instead, fetch all the group memberships. (There won't be that many for our
-      // users anyway.)
-      for (ManagedGroupAccessResponse group : endUserGroupsApiProvider.get().getGroups()) {
```

Comment:
This appears to have been the only usage of `endUserGroupsApiProvider`, should be able to remove

Reply:
Yes – noted in my comment above. I think this will involve enough refactoring that I'd prefer to handle in a follow-up PR. Filed RW-2353 to track.